app/controllers/sessions_controller.rb

```diff
@@ -109,6 +109,10 @@ def login_failure(message)
     render "sessions/new", status: :unauthorized
   end
 
+  def webauthn_failure
+    invalidate_mfa_session(@webauthn_error)
+  end
```

Comment:
Another slightly annoying difference between all the mfa controllers. The session controller fully resets on webauthn failure. I guess this is so you can try again with the right user/password if you have multiple accounts so that your webauthn works?

We should decide what retrying mfa looks like, because destroying the whole mfa session erases the time limit on mfa retry and then we're left with just rack attack :(